src/pages/patterns/forms-pattern/index.mdx

```diff
@@ -592,9 +592,9 @@ Common user errors include:
 <Column colLg={8}>
 <GifPlayer color='dark'>
 
-![Example of client-side error message](/images/inline-error-message.gif)
+![Example of client-side error message](/images/text-input-error.gif)
```

Comment:
This gif is not spanning the full 8 columns.

<img width="907" alt="Screenshot 2024-05-22 at 1 39 53 PM" src="https://github.com/carbon-design-system/carbon-website/assets/43969356/e4d98960-64e0-4fd3-8467-e4c8abdbf5d1">

Reply:
I think there is a build cache issue where the preview isn't updating, seeing if I can force it to update. I just checked locally though and its displaying correctly now.